### PR TITLE
Faster access to ball prediction for Python bots

### PR DESCRIPTION
**Pull request by skyborgff**
Add parameter to team_color in rendering to use enemies color instead default is team color

### CODE REVIEW

**Comment by skyborgff** [comment]
for some reason my last pr comes along as well :(

**Comment by NicEastvillage** [comment]
This is already there https://github.com/RLBot/RLBot/blob/master/src/main/python/rlbot/utils/rendering/rendering_manager.py#L275

**Comment by tarehart** [comment]
We ***definitely*** want "Squash and merge" for this one, there are tons of binaries and broken commits. I'm going to approve this now, and Skyborg feel free to squash and merge when you wake up if it looks good to you.

File: src/main/cpp/RLBotInterface/RLBotInterface/GameFunctions/BallPrediction.cpp [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
 #include "BallPrediction.hpp"
+#include <MessageTranslation\FlatbufferTranslator.hpp>
+#include "..\CallbackProcessor\SharedMemoryDefinitions.hpp"
+#include "..\CallbackProcessor\CallbackProcessor.hpp"
 #include <BoostUtilities\BoostUtilities.hpp>
 #include <BoostUtilities\BoostConstants.hpp>
 
 namespace BallPrediction
-{
+{
```
Comment (dtracers) [inline src/main/cpp/RLBotInterface/RLBotInterface/GameFunctions/BallPrediction.cpp]: remove the spaces at the end of the lines

File: src/main/python/rlbot/utils/structures/ball_prediction_struct.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+MAX_SLICES = 3600
+
+
+class Slice(ctypes.Structure):
+    _fields_ = [("physics", Physics)]               # ("physics", Physics),
+
+
+class BallPrediction(ctypes.Structure):
+    _fields_ = [("slices", Slice * MAX_SLICES),    # ("game_cars", PlayerInfo * MAX_PLAYERS),
```
Comment (dtracers) [inline src/main/python/rlbot/utils/structures/ball_prediction_struct.py]: the comments are wrong

**Pull request merged**